tutorials/colors/colormap-manipulation.py

```diff
@@ -255,6 +255,43 @@ def plot_linearmap(cdict):
 
 plot_examples([cmap1, cmap2])
 
+#############################################################################
+# .. _reversing-colormap:
+#
+# Reversing a colormap
+# ====================
+#
+# Any colormap can be reversed using `.Colormap.reversed`. For example:
+
+colors = ["#ffffcc", "#a1dab4", "#41b6c4", "#2c7fb8", "#253494"]
+my_cmap = ListedColormap(colors, name="my_cmap")
+
+my_cmap_r = my_cmap.reversed()
+
+plot_examples([my_cmap, my_cmap_r])
+# %%
+# This method reverses the parent colormap. If no name is passed in, it also
```

Comment:
"parent" is a bit unclear here. Typically, "parent" is used for parent-child relations, but the colormaps are not related (linked hierarchially). I think of `reversed()` rather as a transformation that creates a new object. I think with the proposed change above ("creates a reversed copy") we are very explicit and you can leave out the parent terminology.

Reply:
"original colormap"?